curriculum/test/test-challenges.js

```diff
@@ -156,8 +156,13 @@ async function getChallenges(lang) {
 }
 
 function validateBlock(challenge) {
-  const dashedBlock = dasherize(challenge.block);
-  if (!helpCategory.hasOwnProperty(dashedBlock)) {
+  const { block, title } = challenge;
+  const dashedBlock = dasherize(block);
+  const isProjectCurriculumChallenge = title.match(/^Part\s*\d+/);
+  if (
+    !helpCategory.hasOwnProperty(dashedBlock) &&
+    !isProjectCurriculumChallenge
```

Comment:
I think it would be better to add the block to `helpCategory`.  Otherwise all the new challenges will be sent to the generic 'Help' category, which probably isn't the best place for them.

Reply:
@ojeytonwilliams, thank you for taking a look at this. You're right, it's better to add the block here so users can create forum posts in the relevant category for each project. I'll add these as soon as possible.